modules/environment/login/default.nix

```diff
@@ -85,8 +85,8 @@ in
       prootStatic =
         let
           crossCompiledPaths = {
-           aarch64 = "/nix/store/45jqhgc0cg1l6vrl95p3qg3j4wh6w59y-proot-termux-aarch64-unknown-linux-android-unstable-2021-11-21";
-           i686 = "/nix/store/8nww78ndkbxqhi4h9f0ly9zkwixa0cd9-proot-termux-i686-unknown-linux-android-unstable-2021-11-21";
+           aarch64 = "/nix/store/dapbgzbpl426jrhz4a2sdl096a8l98ad-proot-termux-aarch64-unknown-linux-android-unstable-2021-11-21";
+           i686 = "/nix/store/6nnjhrh8pgyxjnya72irahxpkbnxai1w-proot-termux-i686-unknown-linux-android-unstable-2021-11-21";
```

Comment:
Out of curiosity: Why did these change?

Reply:
I don't know, actually. The -O3 I've added isn't to blame.

Reply:
It isn't? It is a change in the inputs of the derivation and (unless being the default) changes the output.

(Didn't see the `-O3` there before)

Reply:
Yeah,  but I couldn't figure out what have I changed...

Reply:
I'm not sure I understand what you mean.. If you add `CFLAGS`, this is what you changed. :D

Reply:
I meant, it changed before I've added CFLAGS, and the best explanation I have is the renaming of the patch.

Reply:
Ah gotcha. Yeah that also sounds reasonable, because you are using the path not the content of the patch file as derivation input.